### PR TITLE
vgmstream r1050-3312-g70d20924

### PR DESCRIPTION
**Pull request by xxyzz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Please format your commit as the PR title

**Comment by jonchang** [comment]
We have autosquash now, so there's not really a need for amending and rebasing except in certain cases.

**Comment by jonchang** [comment]
```
==> brew audit vgmstream --online --git --skip-style
==> FAILED
Error: 1 problem in 1 formula detected
vgmstream:
  * stable version should not decrease (from 405509 to 20924)
```

Looks like upstream doesn't believe in version numbers. You'll need to set `version_scheme 1` and manually specify a `version` (perhaps `r1050-3312`)?

**Comment by xxyzz** [comment]
Why `brew audit --strict` doesn't cover all the CI test?

**Comment by jonchang** [comment]
That one would have been covered by `brew audit --strict --git`. We should probably update the pull request template at some point.

File: Formula/vgmstream.rb [hunk 1]
```diff
@@ -21,9 +23,8 @@ class Vgmstream < Formula
   depends_on "mpg123"
 
   def install
-    system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", "."
-    system "make", "vgmstream_cli"
-    system "make", "vgmstream123"
+    system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", "-DCMAKE_BUILD_TYPE=Release", "."
```
Comment (SMillerDev) [inline Formula/vgmstream.rb:26]:
```suggestion
    system "cmake", "-DBUILD_AUDACIOUS:BOOL=OFF", *std_cmake_args, "."
```

**Review by SMillerDev — APPROVED** [review]
Thanks @xxyzz  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**